### DISCUSSION TITLE
Jeepney TV

### DISCUSSION THREAD

**Discussion by thedarkdragon11** [iptv-org/community#669] [Q&A]
**Channel Name:** Jeepney TV
**Channel ID:** JeepneyTV.ph
**Country:** Philippines
**Language:** Tagalog
**External Link (optional):** xxx
**Notes (optional):** Classic shows channel in the Philippines.

**Comment by freearhey** [comment]
Closed in favor of https://github.com/iptv-org/iptv/issues/19667
